engine/runtime-instrument-common/src/main/java/org/enso/interpreter/instrument/command/SetExecutionEnvironmentCommand.java

```diff
@@ -81,11 +80,11 @@ private void setExecutionEnvironment(
                           return null;
                         });
               } else {
-                logger.log(
-                    Level.FINE,
-                    "Requested environment '{}' is the same as the current one. Request has no"
```

Comment:
This must have been broken as `java.util.logging.Logger` doesn't recognize `{}` as a variable placeholder.